andela/vale-ah-backend
# #164139782 Pagination Support for Recipes

#### What does this PR do?
- adds pagination support for listing recipes
#### Description of Task to be completed?
- write unit tests
- accept `page` and `limit` query parameters from request url
- validate query parameters with `Joi`
- document feature with swagger
#### How should this be manually tested?
- ensure build is passing on Travis
- send a *GET* request to `/api/recipes` and pass in `page` and/or `limit` query params
#### Any background context you want to provide?
Pagination support is lacking for our recipes. Reading can be a nightmare when we have so many recipes.
#### What are the relevant pivotal tracker stories?
[#164139782](https://www.pivotaltracker.com/story/show/164139782)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
